.github/workflows/upload_image.yml

```diff
@@ -1,5 +1,6 @@
 name: Upload Image
 on:
+  workflow_dispatch: {}
```

Comment:
Does this mean we may need to trigger this workflow manually in the future?

Reply:
It means we "Could", not "Need". It gives a way to build the image manually, (for debugging/profiling CI or other purposes.)

I still keep the 
```
on:
  release:
    types: [published]
```
this trigger will be triggered by tagging.

Reply:
👌🏽Got it.